sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -903,6 +903,12 @@ object RapidsConf {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(1024 * 1024 * 1024)
 
+  val SHUFFLE_UCX_ACTIVE_MESSAGES_MODE = conf("spark.rapids.shuffle.ucx.activeMessages.mode")
+    .doc("UCX Active Messages can be RNDV, EAGER, or automatic. We set RNDV by default " +
```

Comment:
Should "automatic" be capitalized in this description for consistency with the others?

Reply:
I reworded this doc in latest commit.